### PR TITLE
Add endpoint to return a random phrase

### PR DESCRIPTION
**Pull request by ricardoruwer**
### Motivation
We need an endpoint to return a random phrase to be displayed on the website.

### Proposed solution
Create it!! :)

### Reviewers
@elainenaomi @mracos @feliperenan @odineiramone @charlesoft @caiquemitsuoka

### CODE REVIEW

**Comment by ricardoruwer** [comment]
> Some changes are about parentheses around method function calls. This chages are really needed? Isn't better create a separated commit for them?

@odineiramone It happens automatically in my editor when saving the file 😸
It's because of the `mix format`.

File: lib/caiquenator_web/controllers/phrase_controller.ex [hunk 1]
```diff
@@ -0,0 +1,13 @@
+defmodule CaiquenatorWeb.PhraseController do
+  use CaiquenatorWeb, :controller
+
+  @phrases File.cwd!()
```
Comment (amandasposito) [inline lib/caiquenator_web/controllers/phrase_controller.ex]: You can use `@external_resource` here (https://hexdocs.pm/elixir/Module.html#module-external_resource) this way when the file is changed you won't need to recompile the application.

Comment (amandasposito) [reply lib/caiquenator_web/controllers/phrase_controller.ex]: it was an @josevalim tip xD

Comment (ricardoruwer) [reply lib/caiquenator_web/controllers/phrase_controller.ex]: Done. Can you check if I did it right? 😆

**Pull request merged**